### PR TITLE
[FIX] quality_control_mrp: partial fabrications...

### PR DESCRIPTION
**Pull request by oihane**
... where creating more inspections than required

### CODE REVIEW

File: quality_control_mrp/models/mrp_production.py [hunk 1]
```diff
@@ -24,13 +24,15 @@ def _count_inspections(self):
...
+        done_moves = production.move_created_ids2.filtered(
+            lambda r: r.state == 'done')
         res = super(MrpProduction, self).action_produce(
             production_id, production_qty, production_mode, wiz=wiz)
         if production_mode == 'consume_produce':
             inspection_model = self.env['qc.inspection']
-            production = self.browse(production_id)
             for move in production.move_created_ids2.filtered(
-                    lambda r: r.state == 'done'):
+                    lambda r: r.state == 'done' and r not in done_moves):
```
Comment (pedrobaeza) [inline quality_control_mrp/models/mrp_production.py]: Better to use difference operator: `for move in production.move_created_ids2.filtered(lambda r: r.state == 'done') - done_moves:`

**Comment by pedrobaeza** [comment]
:+1: Thanks for the extra test!

**Comment by oihane** [comment]
After taking a look to the travis error, it is not an error caused by the module in this PR. So please more reviews...

**Comment by CarlosMolinaP** [comment]
:+1: Good Job!!

**Pull request merged**